python/mxnet/optimizer/optimizer.py

```diff
@@ -502,6 +545,7 @@ def __init__(self, momentum=0.0, lazy_update=True, **kwargs):
         super(SGD, self).__init__(**kwargs)
         self.momentum = momentum
         self.lazy_update = lazy_update
+        self.aggregate_num = int(os.getenv('MXNET_OPTIMIZER_AGGREGATION_SIZE', "4"))
```

Comment:
in line 510 can you add a section on aggregate updates and in line 524 can also point to these two methods - `multi_sgd_mom_update` and `multi_mp_sgd_update` as optimizer update rules.

Reply:
Will do.

Reply:
I still think the 'multi' update methods should show up in the SGD doc description. But I am okay with the code owner/merger making a call on this.